lab-michael/app/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
You need to be sure to clean up the indentation in your code. It's gnarly as is...